ZenPacks/zenoss/OpenStackInfrastructure/zenopenstack.py

```diff
@@ -897,6 +927,79 @@ def doTask(self):
             self.state = TaskStates.STATE_IDLE
 
 
+class OpenStackVnicTask(BaseTask):
```

Comment:
Just a comment: seems like we're putting a lot of extraneous code in zenopenstack.

Reply:
What do you mean?

Reply:
I think of the services like zenpython and zenopenstack as generic collection services. It just seems like there is a lot of model-specific code going into zenopenstack.